src/benchmarks/micro/runtime/StoreBlock/StoreBlock.tt

```diff
@@ -0,0 +1,114 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ output extension=".cs" #>
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+[BenchmarkCategory(Categories.Runtime, Categories.JIT)]
+public class StoreBlock
+{
+    const int OperationsPerInvoke = 1000;
+
+    byte[] _srcData = new byte[1024];
+    byte[] _dstData = new byte[1024];
+
+    [GlobalSetup]
+    public void Setup()
+    {
+        for (int i = 0; i < _srcData.Length; i++)
```

Comment:
These benchmarks might be sensitive to memory alignment so we should allocate the arrays in `[GlobalSetup]` so in the future we can enable memory randomization for it (https://github.com/dotnet/performance/pull/1587). It's relatively new feature that very few ppl are aware of.

```suggestion
    byte[] _srcData;
    byte[] _dstData;

    [GlobalSetup]
    public void Setup()
    {
        _srcData = new byte[1024];
        _dstData = new byte[1024];
        for (int i = 0; i < _srcData.Length; i++)
```

Reply:
Thank you for suggestion - I haven't heard about this feature!